### PR TITLE
Add missing packages for a clean docker build run

### PR DESCRIPTION
**Pull request by August2111**
Brief summary of the changes
----------------------------

Solve Issue #519 'In Docker some packages missing for an error free run'
Add 3 Packages to the docker-provisioning-file: 
* overall build environment: cmake, ninja-build
* linux target: libglm-dev

Related issues and discussions
------------------------------

The same packages are needed for a build in the native environment

### CODE REVIEW

**Comment by August2111** [comment]
Of course I don't need the issue tracking before, if I want to solve such an issue?

File: ide/provisioning/install-debian-packages.sh [hunk 1]
```diff
@@ -46,7 +45,7 @@ apt-get install $APTOPTS make g++ \
   imagemagick gettext \
   mesa-common-dev libgl1-mesa-dev libegl1-mesa-dev \
   fonts-dejavu \
-  xz-utils
+  xz-utils  libglm-dev
```
Comment (MaxKellermann) [inline ide/provisioning/install-debian-packages.sh:48]: This is certainly wrong.

**Comment by MaxKellermann** [comment]
> Of course I don't need the issue tracking before, if I want to solve such an issue?

If you send a PR, you don't need to report an issue.

But I'm not happy with your commit message. It doesn't explain anything.

**Comment by August2111** [comment]
Ok, I explain it much more on the issue site, but you are right, here is it much better...
Compiling XCSoar in the Docker environment it happens 3 stops regarding missing packages:
    * overall build environment: cmake, ninja-build - are missing and at this point the make process is stopping witch a failure, for example to build the c-ares in WIN64 target
    * linux target: libglm-dev - without this installed library I have every time the issue "glm/fwd.hpp" (missing in ui\canvas\opengl\Globals.hpp and in ui\canvas\opengl\Geo.hpp) if I want to compile the UNIX target.
 Maybe there is an other way to solve this, but in my case it was the easiest.

**Comment by August2111** [comment]
Hopefully it is more clear now...

**Comment by MaxKellermann** [comment]
GLM is still wrong. It's used as a git submodule, and thus the dev package is not needed. If you need it, you're doing it wrong.

**Comment by August2111** [comment]
Ok, that's right...
I didn't get some submodule experience up to now ;-(
But it is very  strange, that XCSoar use so much possibilities to integrate 3rd party software in their package!

**Comment by MaxKellermann** [comment]
It's documented in the developer manual.

**Comment by August2111** [comment]
Now I had read this ;-)
But why you use such different methods?

I'm fighting a little bit with GitHub, in some minutes I will make an update of this PR!

**Pull request merged**